### ISSUE TITLE
Logging in Caedarva Mire is broken.

### ISSUE THREAD

**Issue by Serephim** [EdenServer/community#1631]
### Checklist

- [ x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ x] I have provided my client version in the details. (type /ver into your game window)

### Details
Version 30181205_0

Only 2 logging points appear. One at I-7 and one at J-8. Once you collect from them they disappear forever and do not respawn again until a zone or server reset. Happens every time I go there to log.

**Comment by GalkaZajin** [comment]
I was just logging there last week and it did seem very off, i was able to get those same 2 points maybe 6 or 7 times though

**Comment by Serephim** [comment]
Yeah, something is messed up, the first time I went there the first point I hit was a power node with 3 nodes in 1, then the other was normal. Then they respawned in the same place again, then they wouldn't respawn at all. The last 3 times I went I logged both points once then they didn't respawn for like 30 minutes so I left.

**Comment by rpcon** [comment]
Came to post the issue and found it already existed, and still happening.

**Comment by perkuprightmeow** [comment]
Just checked every potential logging point here due to a GM ticket -- all were empty.
